.travis.yml

```diff
@@ -6,14 +6,13 @@ branches:
   only:
     - master
     - starter
-    - issue-9-travis-pytest
-    - issue-11-add-Black-formatter
+    - /.*/
```

Comment:
Please remove this before landing btw. I think the reason you have to do this is because your original patch had it and after removing it, it won't run on the PR anymore because it's still trying to run on your branch.